docs/UserGuide.md

```diff
@@ -1,33 +1,59 @@
-# User Guide
+# SHOCO v1.0 - User Guide
 
 ## Introduction
 
-{Give a product intro}
+SHOCO is a command-line interface (CLI) application that allows users to 
+manage and plan their shopping lists and budget. If you are a fast typer, 
+you will find SHOCO to be even more effective than applications with graphical 
+user interfaces.
```

Comment:
thanks for adding in the introduction for the team!

Reply:
No worries!